meeting-notes/11-28February2019.md

```diff
@@ -0,0 +1,107 @@
+# dat protocol Community communication
+
+28 Feb 2019
+
+## People
+
+* Diego ([@dpaez](https://github.com/dpaez))
+* Joseph ([ARA](https://ara.one/))
+* Georgy ([@rangermauve](https://github.com/rangermauve))
+
+## Agenda
+
+* No previous action items
+
+## Meeting Notes
+
+- Diego starts introducing a new comm-comm meeting, the number 11.
+  Comm-comm meetings are a place for and by the community. Here we can
+  discuss ideas, share updates and findings, answer community questions,
+  and much more. This place is yours.
+- Georgy says hi. Since Georgy is working full time for the Dat project he
```

Comment:
```suggestion
- Georgiy says hi. Since Georgiy is contracting part time for the Dat project he
```

I'm working 20h a week and I'm technically an independent contractor. :D